source/opt/inline_pass.cpp

```diff
@@ -0,0 +1,430 @@
+// Copyright (c) 2017 The Khronos Group Inc.
+// Copyright (c) 2017 Valve Corporation
+// Copyright (c) 2017 LunarG Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "inline_pass.h"
+
+// Indices of operands in SPIR-V instructions
+
+static const int kSpvEntryPointFunctionId = 1;
+static const int kSpvFunctionCallFunctionId = 2;
+static const int kSpvFuncitonCallArgumentId = 3;
+static const int kSpvFunctionParameterResultId = 1;
+static const int kSpvReturnValueId = 0;
+static const int kSpvTypePointerStorageClass = 1;
+static const int kSpvTypePointerTypeId = 2;
+
+namespace spvtools {
+namespace opt {
+
+uint32_t InlinePass::FindPointerToType(uint32_t typeId) {
```

Comment:
What would you want to call the method?

Another option is delete this method and just make the call

... = GetId(Pointer(GetType(type_id), SpvStorageClassFunction));

Which would you prefer?